data/translations/ru/register.json

```diff
@@ -0,0 +1,37 @@
+  "state_heading": "Регистрация избирателей в %state_name%",
+  "by_mail__intro": "Онлайн-регистрация в настоящее время недоступна. Узнайте, как зарегистрироваться для голосования на  %link%",
+  "by_mail__link": "веб-сайте выборов в штате %state_name% ",
+  "other_language_selection__default": "Выберите Язык",
+  "other_language_selection__heading": "Другие способы регистрации для голосования в %state_name%",
+  "other_language_selection__label": "Вы также можете загрузить Национальную регистрационную форму избирателя (PDF), доступную на английском и 14 дополнительных языках, из раскрывающегося меню ниже.",
+  "other_language_selection__submit": "Загрузите национальную регистрационную форму избирателя",
+  "in_person__intro": "Зарегистрируйтесь лично в своем местном избирательном участке. Узнайте больше о том, как зарегистрироваться на %link%",
+  "in_person__link": "веб-сайте выборов в штате %state_name%",
+  "not_needed__intro": "Регистрация избирателей не требуется в %state_name%. Узнайте подробнее о голосовании на %link%",
+  "not_needed__link": "избирательном веб-сайте %state_name%",
+  "online__answer1": "Начните онлайн-регистрацию на веб-сайте %link%",
+  "online__answer1_link": "выборов штата %state_name%",
+  "online__answer2": "Вы также можете зарегистрироваться для голосования по почте или лично на веб-сайте %link%",
+  "online__answer2_link": "выборов штата %state_name% ",
+  "online__h2_question2": "Другие способы регистрации для голосования в %state_name%",
+  "online__h2_question1": "Как зарегистрироваться для голосования в %state_name%",
+  "alert_state__alert_msg": "Из-за COVID-19 могут произойти изменения в том, когда, где и как вы голосуете на предстоящих выборах. Для получения дополнительной информации свяжитесь с %link%. ",
+  "alert_state__alert_msg1": "веб-сайтом выборов вашего штата",
+  "alert_state__alert_after_election": "Для получения официальных результатов выборов перейдите по ссылке (веб-сайта вашего штата)",
+  "dates__bymail_deadline": "Зарегистрируйтесь по почте в срок:",
+  "dates__byonline_deadline": "Крайний срок регистрации он-лайн:",
+  "dates__electionday": "День Всеобщих Выборов: 3 ноября, 2020.",
+  "dates__electionday_heading": "День Всеобщих Выборов",
+  "dates__heading": "Крайний срок регистрации он-лайн в %state_name%",
+  "dates__inperson_deadline": "Крайний срок личной регистрации он-лайн:",
+  "confirm_registration__heading": "Как проверить вашу регистрацию избирателя",
+  "confirm_registration__intro": "Вы можете подтвердить свой статус избирателя на %link% ",
+  "confirm_registration__link": "веб-сайте штата %state_name%",
+  "confirm_registration__intro": "Вы можете подтвердить свой статус регистрации избирателя, связавшись с местным регистрационным офисом. %link%.",
+  "confirm_registration__link": "Нажмите здесь, чтобы просмотреть контактную информацию клерка округа Вайоминг (PDF)",
+  "confirm_registration__url": "https://sos.wyo.gov/Elections/Docs/WYCountyClerks.pdf?ref=voteusa_en"
```

Comment:
These properties have the wrong keys
```
  "confirm_registration__intro_WY": "Вы можете подтвердить свой статус регистрации избирателя, связавшись с местным регистрационным офисом. %link%.",
  "confirm_registration__link_WY": "Нажмите здесь, чтобы просмотреть контактную информацию клерка округа Вайоминг (PDF)",
```
Delete the property: `confirm_registration__url`